tests/test_forgot_password.py

```diff
@@ -25,14 +27,16 @@ def test_valid_email(self, create_driver, prep_properties):
         about_page.click_login_link()
         login_page.click_forgot_password()
         forgot_password_page.send_password_reset_link(email)
-        assert_that(self._success_msg).is_equal_to(forgot_password_page.get_success_msg())
+        assert_that(self._SUCCESS_MSG).is_equal_to(forgot_password_page.get_success_msg())
 
     @allure.description("Forgot Password feature test with invalid email address")
     def test_invalid_email(self, create_driver):
         about_page = AboutPage()
         login_page = LoginPage()
         forgot_password_page = ForgotPasswordPage()
+        excel_reader = ExcelParser(self._DATA_FILE_NAME)
+        emails = excel_reader.read_from_excel("Emails")
         about_page.click_login_link()
         login_page.click_forgot_password()
-        forgot_password_page.send_password_reset_link("something@gmail.com")
-        assert_that(self._error_msg).is_equal_to(forgot_password_page.get_invalid_email_msg())
+        forgot_password_page.send_password_reset_link(emails[0])
```

Comment:
can we make this parametrized? like DDT from excel? 

I want 1 example like this and 1 like DDT